src/mscorlib/src/System/Globalization/CultureData.cs

```diff
@@ -308,6 +309,10 @@ private static StringStringDictionary RegionNames
         private static volatile StringCultureDataDictionary s_cachedRegions;
         private static volatile StringStringDictionary s_RegionNames;
 
+        private const string c_InvariantModeConfigSwitch = "System.Globalization.Invariant";
+        private static bool s_invariantGlobalizationMode;
+        internal static bool InvariantMode { get { return s_invariantGlobalizationMode; } }
```

Comment:
Subsequent to this change, should we consider exposing this publicly from somewhere?  I could see someone wanting to check it for verification purposes.